server/services/store/sqlstore/migrate.go

```diff
@@ -217,15 +221,23 @@ func (s *SQLStore) Migrate() error {
 		return err
 	}
 
-	if err := ensureMigrationsAppliedUpToVersion(engine, driver, uniqueIDsMigrationRequiredVersion); err != nil {
+	if err := s.ensureMigrationsAppliedUpToVersion(engine, driver, uniqueIDsMigrationRequiredVersion); err != nil {
 		return err
 	}
 
 	if err := s.runUniqueIDsMigration(); err != nil {
 		return fmt.Errorf("error running unique IDs migration: %w", err)
 	}
 
-	if err := ensureMigrationsAppliedUpToVersion(engine, driver, categoriesUUIDIDMigrationRequiredVersion); err != nil {
+	if err := s.ensureMigrationsAppliedUpToVersion(engine, driver, teamLessBoardsMigrationRequiredVersion); err != nil {
```

Comment:
Moved teamless migration (requires version 18) to happen before categories UUID migration (requires version 20). @harshilsharma63 please confirm that this is ok